CHANGELOG.md

```diff
@@ -17,8 +17,8 @@
 
 - Fix video replay crashes due to video writer inputs not marked as finished on cancellation (#5608)
 - Fix wrong flush timeout (#5565). When flush timed out before the SDK finished sending data, it always blocked the full flush timeout the next time being called. This is fixed now.
-
 - Launch profiling now respects original configured options if they change on the next launch (#5417)
+- Ignore frame delays when going into background (#5681)
```

Comment:
`m`: This PR fixes getting wrong frame statistics when the app goes to the background and again to the foreground which has the positive side effect of fixing wrongly reported app hangs when the app was in the background only for app hangs v2, because app hangs v2 uses the frame statistics.
The changelog entry and the PR title should reflect that impact.